### PR TITLE
INS-3753 [Invoice] Delete invoice endpoint

### PR DESCRIPTION
**Pull request by ShakirovaE**

### CODE REVIEW

File: sell/sell-api.yaml [hunk 1]
```diff
@@ -3059,6 +3059,24 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+    delete:
+      tags:
+        - Invoices
+      summary: Delete an invoices
```
Comment (Zemidar) [inline sell/sell-api.yaml:3065]:
```suggestion
      summary: Delete the invoice
```

File: sell/sell-api.yaml [hunk 2]
```diff
@@ -3059,6 +3059,24 @@ paths:
...
+      description: Use this endpoint to remove an invoice.
```
Comment (Zemidar) [inline sell/sell-api.yaml:3066]:
```suggestion
      description: Use this endpoint to remove invoice by id.
```

**Review by Zemidar — APPROVED** [review]

File: sell/sell-api.yaml [hunk 3]
```diff
@@ -3059,6 +3059,24 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+    delete:
+      tags:
+        - Invoices
+      summary: Delete the invoice
+      description: Use this endpoint to remove invoice by id.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:3066]:
```suggestion
      description: Use this endpoint to remove the invoice by its ID. Note that the payments associated with this invoice remain in the system.
```

**Review by olyavertwist — CHANGES_REQUESTED** [review]
Small update for better user experience

**Review by olyavertwist — APPROVED** [review]

**Pull request merged**